hooks/build

```diff
@@ -143,12 +147,33 @@ echo
 
 # Extract python version from image name we want to build in case it was passed
 # Via IMAGE_NAME
-# This nice bash construct below extracts last field after '-' delimiter
-# So for example 'latest-3.6' will produce PYTHON_VERSION=3.6
-PYTHON_VERSION=$(echo "${LOCAL_IMAGE}" | rev | cut -d '-' -f1 | rev )
-export PYTHON_VERSION
+# This nice bash construct below extracts last field after '-python' delimiter
+# So for example 'airflow:master-python3.6' will produce LONG_PYTHON_VERSION=python3.6
+LONG_PYTHON_VERSION=$(echo "${LOCAL_IMAGE}" | rev | cut -d '-' -f1 | rev )
+export LONG_PYTHON_VERSION
+
+echo
+echo "LONG_PYTHON_VERSION=${LONG_PYTHON_VERSION}"
+echo
+
+if [[ ! ${LONG_PYTHON_VERSION} =~ python[2-3]\.[0-9]+ ]]; then
+    echo
+    echo "ERROR! Python version extracted from IMAGE_NAME does not match the pythonX.Y format"
```

Comment:
```suggestion
    echo >&2 "ERROR! Python version extracted from IMAGE_NAME does not match the pythonX.Y format"
```

etcfor these lines

Reply:
Yep! I will look for other similar lines!